modules/installation-aws-upi-bootstrap.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_aws_upi/installing-aws-upi.adoc
+
+[id="installation-aws-upi-bootstrap-{context}"]
+= Initializing the bootstrap node on AWS UPI
+
+After you create all of the required infrastructure in Amazon Web Services (AWS),
+you can install the cluster.
+
+.Prerequisites
+
+* Configure an AWS account.
+* Generate the ignition config files for your cluster.
+* Create and configure a VPC and assocated subnets in AWS.
+* Create and configure DNS, load balancers, and listeners in AWS.
+* Create control plane and worker roles.
+* Create the bootstrap node.
```

Comment:
The bootstrap machine/instance/whatever is not a node in the Kubernetes sense, since it is not available to the control-plane for scheduling Pods.  Can we say "bootstrap machine" or "bootstrap host" or something more generic?  I feel like the "machine" wording has come up before, so feel free to mark this resolved if the conclusion was "'node' is the best we can do" ;).

Reply:
@wking, it has, and I think that if "machine" is really the right term, then it can stay. (I'm lumping it in the same category as "desired" - the style guide says "no," but Kubernetes trumps it.)